packages/gittools.rb

```diff
@@ -3,11 +3,12 @@
 class Gittools < Package
   description 'Tools for analyzing .git repositories'
   homepage 'https://github.com/internetwache/GitTools'
-  version '24eaef0d11e590643e65d188b017b49414d81cc2'
+  @commit = '24eaef0d11e590643e65d188b017b49414d81cc2'
+  version @commit[0...7]
   license 'MIT'
   compatibility 'all'
   source_url 'https://github.com/internetwache/GitTools.git'
-  git_hashtag version
+  git_hashtag @commit
```

Comment:
I'm conscious of the fact that this is still quite similar to using @_ver-- is there any other way to do this which doesn't introduce an additional variable?